web-api/src/storage/elastic.rs

```diff
@@ -404,6 +409,15 @@ impl Client {
     }
 }
 
+pub(super) trait SerializeDocumentIds: Serialize {}
+impl<T> SerializeDocumentIds for &'_ T where T: SerializeDocumentIds {}
```

Comment:
:art: this could be generalized a bit
```suggestion
impl<T> SerializeDocumentIds for &'_ T where T: SerializeDocumentIds + ?Sized {}
```

Reply:
`T: Serialize` is implied which also implies `Sized`